packages/grid/x-data-grid/src/tests/quickFiltering.DataGrid.test.tsx

```diff
@@ -372,4 +400,40 @@ describe('<DataGrid /> - Quick Filter', () => {
       expect(getRows({ quickFilterValues: [''] }).year).to.deep.equal(ALL_ROWS_YEAR);
     });
   });
+
+  // See https://github.com/mui/mui-x/issues/6783
+  it('should not override user input when typing', async function test() {
+    if (isJSDOM) {
+      this.skip();
+    }
+    // Warning: this test doesn't fail consistently as it is timing-sensitive.
```

Comment:
While this test doesn't consistently fail without the fix, I thought it was still better than nothing 🙂
What do you think?

Reply:
Not ideal but it's fine as it is.